public/js/components/Filters/Filters.js

```diff
@@ -20,7 +20,9 @@ class Filters extends React.Component {
                                     <select
                                         className="form__field form__field--select"
                                         onChange={event => onSelectChange({
-                                            office: event.target.value
+                                            office: event.target.value,
+                                            startDate: filterVals.startDate,
```

Comment:
As discussed, worth doing a bit of research on best-practice of getting/passing state in async places, so we can come back to this one 👍